Source/Fuse.Controls.Primitives/TextControls/TextControl.Properties.uno

```diff
@@ -125,10 +125,18 @@ namespace Fuse.Controls
 		{
 			get
 			{
-				if (FontScale > 0)
```

Comment:
I don't think this is a good way to implement the feature. It is common convention in Uno/C# that properties should read and write as the same value. This will work very poorly with e.g. visual tooling. 

Transformations should be done through a separate property, e.g. `FontSizeScaled` which is `get`-only. Then do a search for all call sites of of `FontSize` and evaluate per call site whether to read `FontSizeScaled` or `FontSize`.

Reply:
I've made changed as per suggestions

Reply:
Good stuff, but see my last comment